litmus/director/TCID-DIR-OP-INSTALL-OPENEBS-DP-ON-SPECIFIC-NODE/test.yml

```diff
@@ -198,6 +203,45 @@
             set_fact:
               openebs_id: "{{ install_openebs.json.id }}"
 
+          - name: Labeling the nodes of the cluster
+            uri:
+              url: "{{ director_url }}/v3/groups/{{ group_id }}/openebses/{{ openebs_id }}/?action=selectopenebsnodes"
+              method: POST
+              url_username: "{{ username.stdout }}"
+              url_password: "{{ password.stdout }}"
+              force_basic_auth: yes
+              return_content: yes
+              body_format: json
+              body: '{"dataPlaneNodes":["{{node1_id}}"]}'
+              status_code: 200
+            register: labelnodes
+
+          - name: Labeling the nodes of the cluster
```

Comment:
Rename the task to - `Labeling second node of the cluster as control plane node`